internal/stateindex/encoding.go

```diff
@@ -81,3 +98,35 @@ func decodeReverseOrderVarUint64(b []byte) uint64 {
 
 	return math.MaxUint64 - v
 }
+
+const hextable = "0123456789abcdef"
```

Comment:
`const` are usually at the top of the file, no?

Reply:
Done